### DISCUSSION TITLE
关于秒传遇到的20GB limit 限制

### DISCUSSION THREAD

**Discussion by HamBosco** [AlistGo/alist#7486] [Q&A] [answered]
请问20GB limit限制是最近加的吗？有了这个限制，基本原盘都没办法秒传别的网盘了。

**Comment by pongfcnkl** [comment, marked as answer]
beta已经修复了，可以自己下载替换，正式版等发版
https://github.com/AlistGo/alist/commit/2671c876f1fe7e6de1a3939c9ee5f588d9b5f41b
